src/dso_api/dynamic_api/routers.py

```diff
@@ -68,6 +68,11 @@ def _initialize_viewsets(self) -> List[Type[DynamicModel]]:
         # This makes sure the 'to' field is resolved to an actual model class.
         for app_label, models_by_name in self.all_models.items():
             for model in models_by_name.values():
+                # Register model in Django apps.
+                apps.register_model("datasets", model)
+                if model._table_schema.get("schema", {}).get("parentTable") is not None:
```

Comment:
wellicht `model._has_parent_table` of andere functie? Dit is wat mij betreft iets teveel in de internals prikken.